### PR TITLE
Updated strings from Crowdin 20221202

### PR DESCRIPTION
**Pull request by mikehardy**
First i18n sync for a while - goal is to clear translations prior to merge of #12765

### CODE REVIEW

File: AnkiDroid/src/main/res/values-fa/01-core.xml [hunk 1]
```diff
@@ -175,7 +175,7 @@
     <string name="sd_card_not_mounted">حافظۀ دستگاه یافت نشد</string>
     <string name="empty_cloze_warning">این یادداشت را ویرایش و چند کلوز اضافه کنید. (%1$s)</string>
     <string name="button_sync" comment="Text of the sync button">همگام سازی</string>
-    <string name="button_sync_disabled" comment="Text of the sync button when disabled">Sync temporarily disabled</string>
+    <string name="button_sync_disabled" comment="Text of the sync button when disabled"></string>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-fa/01-core.xml:178]: Nope

Comment (mikehardy) [reply AnkiDroid/src/main/res/values-fa/01-core.xml:178]: fixed, needs resync on or after 12:53

**Review by BrayanDSO — APPROVED** [review]
LGTM on a quick scan

**Review by mikehardy — COMMENTED** [review]
good to go - only change was me removing a busted translation, so no CI needed

**Pull request merged**